src/it/resources/expected/all_datatypes/cpp-headers/all_datatypes.hpp

```diff
@@ -1,5 +1,5 @@
 // AUTOGENERATED FILE - DO NOT MODIFY!
-// This file generated by Djinni from all_datatypes.djinni
+// This file was generated by Djinni from all_datatypes.djinni
```

Comment:
This is the change in the Djinni header that I did. It affects every single output file for the integration tests. I hope you agree that the sentence was not correct before this change. Sorry for bloating the PR with it, it's really just a small change that I had to fix once I saw it. 🙈